dead.py

```diff
@@ -200,28 +232,17 @@ def main(argv: Optional[Sequence[str]] = None) -> int:
     files_re = re.compile(args.files)
     exclude_re = re.compile(args.exclude)
     tests_re = re.compile(args.tests)
-    for filename, is_test in _filenames(files_re, exclude_re, tests_re):
-        tree = _ast(filename)
 
-        with visitor.file_ctx(filename, is_test=is_test):
-            visitor.visit(tree)
-
-            with open(filename, 'rb') as f:
-                for tp, s, (lineno, _), _, _ in tokenize.tokenize(f.readline):
-                    if tp == tokenize.COMMENT:
-                        visitor.visit_comment(lineno, s)
```

Comment:
why did you change this? all you needed to do was add functionality to `visit_comment` :(

Reply:
it is more convenient to write tests for me